lib/twilio/call.ts

```diff
@@ -851,6 +862,45 @@ class Call extends EventEmitter {
     }
   }
 
+  /**
+   * Send a message to Twilio.
```

Comment:
Need to expand more on this. Just calling it out. No need to address now. We'll do it as part of polishing up the docs.